client/my-sites/posts/post.jsx

```diff
@@ -278,7 +278,7 @@ module.exports = React.createClass({
 
 		if ( showLikes ) {
 			if ( post.like_count > 0 ) {
-				likeTitle = this.translate( '1 Like', '%(count)s Likes', {
+				likeTitle = this.translate( '%(count)s Like', '%(count)s Likes', {
```

Comment:
Actually, this code was causing a warning, but not in the browser console... but rather in the terminal via i18nlint.

![eslint warning](https://cloudup.com/c5QesLzwxY0+)